### ISSUE TITLE
[ERROR] About tampermonkey beta runtime error

### ISSUE THREAD

**Issue by waahah** [Tampermonkey/tampermonkey#2152]
**An error occurred in installation script:**
```js
Unchecked runtime.lastError: The page keeping the extension port is moved into back/forward cache, so the message channel is closed.
```
*This error occurs every time a script is installed, and sometimes another error occurs:*
```js
Uncaught (in promise) Error: No tab with id: 693405998.
```
For example: [https://github.com/Tampermonkey/tampermonkey/issues/2138](https://github.com/Tampermonkey/tampermonkey/issues/2138)

## Suggest
> Disable bfcache and roll back the Enhanced back/forward cache navigation handling function

*Because the browser suspends any pending timer or unresolved promise for the page in bfcache (including almost all pending tasks in the JavaScript task queue) and continues processing tasks after the page recovers from bfcache. This may affect other open tabs in the same source. As a result, browsers typically do not try to cache pages when using apis that might affect other pages during messaging processing.*

## Specifications

- Google Chrome: 126.0.6478.127 (Official Version) (64-bit)
- TM: Tampermonkey BETA 5.3.6206
- OS: Window 10

**Comment by derjanb** [comment]
What is the bug besides these console logs? Tampermonkey should detect if a page is moved in bfcache and restore communication when it is moved out of it.

**Comment by waahah** [comment]
In addition to the above error, no other error has been found，but I think the above problem is a universal one
![屏幕截图 2024-08-09 234923](https://github.com/user-attachments/assets/7ba4dcb2-aa85-491d-ad11-80e06a43274d)

**Comment by derjanb** [comment]
> Unchecked runtime.lastError: The page keeping the extension port is moved into back/forward cache

Where exactly do you see this message? In the console of the script installation page? Can you maybe post a screenshot? 
Do you need to do special things to make it appear?

**Comment by waahah** [comment]
From greasyfork when I click on the install script button, link to jump to [https://www.tampermonkey.net/script_installation.php#url=](https://www.tampermonkey.net/script_installation.php#url=), Service Worker will be output in the above error

![屏幕截图 2024-08-12 160103](https://github.com/user-attachments/assets/f936a85f-05f5-43b8-bb96-2f66f0609279)

**Comment by derjanb** [comment]
> I also didn't noticed any erroneous behavior.

Yes, this is just a cosmetic problem.

**Comment by derjanb** [comment]
Should be fixed at 5.3.6208 ([crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|xpi in review)

Please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions` (after you've enabled 'Developer Mode').

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.
